src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/resource/VisitResourceIntTest.kt

```diff
@@ -554,10 +573,41 @@ class VisitResourceIntTest : IntegrationTestBase() {
         .jsonPath("$.visitorSupport.length()").isEqualTo(updateRequest.supportList!!.size)
         .jsonPath("$.visitorSupport[0].supportName").isEqualTo(updateRequest.supportList!![0].supportName)
         .jsonPath("$.visitorSupport[0].supportDetails").isEqualTo(updateRequest.supportList!![0].supportDetails!!)
-        .jsonPath("$.visitorConcerns").isEqualTo(updateRequest.visitorConcerns!!)
+        .jsonPath("$.visitNotes.length()").isEqualTo(updateRequest.visitNotes!!.size)
+        .jsonPath("$.visitNotes[0].type").isEqualTo(updateRequest.visitNotes!![0].type.name)
```

Comment:
Does this need to check the text has changed?

Reply:
yes for sure... missed that for some reason